### PR TITLE
[improvement] add a config repair_slow_replica

### PR DESCRIPTION
**Pull request by yangzhg**
## Proposed changes

   Add a config `repair_slow_replica`    
    when this config is true, Doris will try to delete the replica
    with the largest number of versions, and then rebalance the replica.
    Usually, when the number of versions of a certain replica is much higher
    than that of other replicas, there are some problems with the current be's compilation.
    Migrating to other machines can typically solve this problem.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [x] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
Please add ut for this case, you can refer to `TabletRepairAndBalanceTest`

**Comment by yangzhg** [comment]
> Please add ut for this case, you can refer to `TabletRepairAndBalanceTest`

done

File: fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java [hunk 1]
```diff
@@ -966,6 +970,39 @@ private boolean handleColocateRedundant(TabletSchedCtx tabletCtx) throws SchedEx
...
+                minVersionCount = replica.getVersionCount();
+                minReplica = replica;
+            }
+        }
+
+        if (chosenReplica != null && !chosenReplica.equals(minReplica) && minReplica.isAlive()) {
+            try {
+                Catalog.getCurrentCatalog().setReplicaStatus(tabletCtx.getTabletId(), chosenReplica.getBackendId(),
+                        Replica.ReplicaStatus.BAD);
+                throw new SchedException(Status.FINISHED, "colocate redundant replica is deleted");
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java:997]:
```suggestion
                throw new SchedException(Status.FINISHED, "set slow replica as bad");
```

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**